### PR TITLE
Update it-IT.lua

### PR DESCRIPTION
**Pull request by Enryy**

### CODE REVIEW

**Review by InZidiuZ — CHANGES_REQUESTED** [review]
There are a lot of missing translations here. There shouldn't be any English within this file.

**Comment by Enryy** [comment]
now it should be ok, moreover other missing translations have been added.

**Comment by InZidiuZ** [comment]
My previous comment still stands. It seems you also added a bunch of random tabs now. Both of these issues needs fixing before a merge.

**Comment by InZidiuZ** [comment]
You did not fix it. It seems to just be getting worse with every 'fix'...

**Comment by Enryy** [comment]
I have rewritten it completely to avoid further problems, if you can point out where there are mistakes so that I can correct them that would be great...

**Comment by InZidiuZ** [comment]
Just CTRL+F and search for two spaces. You'll find some issues just by doing that.

File: languages/it-IT.lua [hunk 1]
```diff
@@ -188,7 +194,7 @@ OP.Global.Locales.Languages["it-IT"] = {
 		wiped_entities = "Entità wipate",
 		wipe_entities_logs_title = "Entità wipate",
 		wipe_entities_logs_details = "${consoleName} problemi con un entità con le seguenti configurazioni: distance = `${distance}`, ignoreLocalEntities = `${ignoreLocalEntities}`, modelName = `${modelName}`",
-
+
```
Comment (InZidiuZ) [inline languages/it-IT.lua]: One example is here...

**Comment by InZidiuZ** [comment]
Please go through your commits properly before pushing them.

**Comment by InZidiuZ** [comment]
Here are some issues:

>   " \n" @ D:/FiveM/not-cool-validator/it-IT.lua:197
  " \n" @ D:/FiveM/not-cool-validator/it-IT.lua:399
  " \n" @ D:/FiveM/not-cool-validator/it-IT.lua:463
  " \n" @ D:/FiveM/not-cool-validator/it-IT.lua:1136
  " \n" @ D:/FiveM/not-cool-validator/it-IT.lua:1397
  " \n" @ D:/FiveM/not-cool-validator/it-IT.lua:1398
  " \n" @ D:/FiveM/not-cool-validator/it-IT.lua:2322
  " \n" @ D:/FiveM/not-cool-validator/it-IT.lua:2340
  " \n" @ D:/FiveM/not-cool-validator/it-IT.lua:2646
  " \n" @ D:/FiveM/not-cool-validator/it-IT.lua:3722
  " \n" @ D:/FiveM/not-cool-validator/it-IT.lua:3807
  " \n" @ D:/FiveM/not-cool-validator/it-IT.lua:4085
  " \n" @ D:/FiveM/not-cool-validator/it-IT.lua:4181
  " \n" @ D:/FiveM/not-cool-validator/it-IT.lua:4492
  " \n" @ D:/FiveM/not-cool-validator/it-IT.lua:4714
  " \n" @ D:/FiveM/not-cool-validator/it-IT.lua:4735
  "\t\n" @ D:/FiveM/not-cool-validator/it-IT.lua:51
  "\t\n" @ D:/FiveM/not-cool-validator/it-IT.lua:616
  "\t\n" @ D:/FiveM/not-cool-validator/it-IT.lua:978
  "\t\n" @ D:/FiveM/not-cool-validator/it-IT.lua:1152
  "\t\n" @ D:/FiveM/not-cool-validator/it-IT.lua:1241
  "\t\n" @ D:/FiveM/not-cool-validator/it-IT.lua:1263
  "\t\n" @ D:/FiveM/not-cool-validator/it-IT.lua:1708
  "\t\n" @ D:/FiveM/not-cool-validator/it-IT.lua:1737
  "\t\n" @ D:/FiveM/not-cool-validator/it-IT.lua:1966
  "\t\n" @ D:/FiveM/not-cool-validator/it-IT.lua:2208
  "\t\n" @ D:/FiveM/not-cool-validator/it-IT.lua:2303
  "\t\n" @ D:/FiveM/not-cool-validator/it-IT.lua:2332
  "\t\n" @ D:/FiveM/not-cool-validator/it-IT.lua:2341
  "\t\n" @ D:/FiveM/not-cool-validator/it-IT.lua:2411
  "\t\n" @ D:/FiveM/not-cool-validator/it-IT.lua:2480
  "\t\n" @ D:/FiveM/not-cool-validator/it-IT.lua:2896
  "\t\n" @ D:/FiveM/not-cool-validator/it-IT.lua:3304
  "\t\n" @ D:/FiveM/not-cool-validator/it-IT.lua:3328
  "\t\n" @ D:/FiveM/not-cool-validator/it-IT.lua:3351
  "\t\n" @ D:/FiveM/not-cool-validator/it-IT.lua:3429
  "\t\n" @ D:/FiveM/not-cool-validator/it-IT.lua:3657
  "\t\n" @ D:/FiveM/not-cool-validator/it-IT.lua:4109
  "\t\n" @ D:/FiveM/not-cool-validator/it-IT.lua:4134
  "\t\n" @ D:/FiveM/not-cool-validator/it-IT.lua:4139
  "\n\n\n" @ D:/FiveM/not-cool-validator/it-IT.lua:2333
  " \t" @ D:/FiveM/not-cool-validator/it-IT.lua:1338
  ".." @ D:/FiveM/not-cool-validator/it-IT.lua:317
  ".." @ D:/FiveM/not-cool-validator/it-IT.lua:804
  ".." @ D:/FiveM/not-cool-validator/it-IT.lua:1605
  ".." @ D:/FiveM/not-cool-validator/it-IT.lua:1610
  ".." @ D:/FiveM/not-cool-validator/it-IT.lua:317
  ".." @ D:/FiveM/not-cool-validator/it-IT.lua:1605
  ".." @ D:/FiveM/not-cool-validator/it-IT.lua:1610
  Trailing comma in table definition @ D:/FiveM/not-cool-validator/it-IT.lua:610
  Trailing comma in table definition @ D:/FiveM/not-cool-validator/it-IT.lua:2123
  Trailing comma in table definition @ D:/FiveM/not-cool-validator/it-IT.lua:2228
  Trailing comma in table definition @ D:/FiveM/not-cool-validator/it-IT.lua:4588
  Trailing comma in table definition @ D:/FiveM/not-cool-validator/it-IT.lua:4593
  Trailing comma in table definition @ D:/FiveM/not-cool-validator/it-IT.lua:4668
  Weird spaces and new-lines @ D:/FiveM/not-cool-validator/it-IT.lua:3806
  Doesn't end with a newline @ D:/FiveM/not-cool-validator/it-IT.lua:496

Note that not all of these are issues, and some of them should be ignored.

File: languages/it-IT.lua [hunk 2]
```diff
@@ -3470,17 +3787,20 @@ OP.Global.Locales.Languages["it-IT"] = {
...
-		roll_message_chat_title = "/lanciadado [${serverId}]",
-		card_message_chat_title = "/carta [${serverId}]",
-		description_message_chat_title = "/descrizione [${serverId}]",
-		message_too_long = "Il messaggio contiene troppi caratteri o linee!"
+		attempt_message_chat_title = "/attempt [${serverId}]",
+		dice_message_chat_title = "/dice [${serverId}]",
+		roll_message_chat_title = "/roll [${serverId}]",
+		card_message_chat_title = "/card [${serverId}]",
+		description_message_chat_title = "/description [${serverId}]",
+		message_too_long = "The message contains too many characters or lines!"
```
Comment (InZidiuZ) [inline languages/it-IT.lua:3798]: ...did you just revert this to English again?

File: languages/it-IT.lua [hunk 3]
```diff
@@ -3470,17 +3787,20 @@ OP.Global.Locales.Languages["it-IT"] = {
...
 	},
 
 	ped_objects = {
 		illegal_ped_object = "Tentativo di aggiungere un oggetto ped alla lista degli oggetti ped 'ammessi'."
+
```
Comment (InZidiuZ) [inline languages/it-IT.lua:3803]: Random newline?

File: languages/it-IT.lua [hunk 4]
```diff
@@ -3569,11 +3890,37 @@ OP.Global.Locales.Languages["it-IT"] = {
...
+		player_not_in_riot_list = "${consoleName} non è nella lista delle rivolte.",
+		added_riot_player = "Aggiunto ${consoleName} alla lista delle rivolte.",
+		failed_to_add_riot_player = "Impossibile aggiungere ${consoleName} alla lista delle rivolte.",
+		removed_riot_player = "Rimosso ${consoleName} alla lista delle rivolte.",
+		failed_to_remove_riot_player = "Impossibile rimuovere ${consoleName} alla lista delle rivolte"
 	},
 
 	screenshots = {
-		screenshot_created = "E' stato creato uno screenshot.",
+	screenshot_created = "E' stato creato uno screenshot.",
```
Comment (InZidiuZ) [inline languages/it-IT.lua:3923]: ...why would you remove a tab from here?

File: languages/it-IT.lua [hunk 5]
```diff
@@ -4515,7 +4918,10 @@ OP.Global.Locales.Languages["it-IT"] = {
 		evidence_type = "Tipo di prova",
 		evidence_casings = "Gli involucri sono tornati al numero seriale ${serialNumber} che era tenuto da ${buyerName} (${buyerCid}) al momento dell'uso.",
 		evidence_bullets = "I danni del proiettile sembrano essere stati creati da ${bulletLabel}.",
-		evidence_dna = "Il DNA e' stato preso dal veicolo con targa ${plateNumber} e tornato indietro a ${DNAOwnerName} (${DNAOwner}).",
+		evidence_car_dna = "Il DNA e' stato preso dal veicolo con targa ${plateNumber} e tornato indietro a ${DNAOwnerName} (${DNAOwner}).",
+		evidence_dna = "DNA collected from ${fullName} #${characterId}.",
+		evidence_fingerprint = "Fingerprint di ${fullName} #${characterId}.",
+		evidence_not_processed = "This evidence bag has not been processed yet.",
```
Comment (InZidiuZ) [inline languages/it-IT.lua:4924]: English.

File: languages/it-IT.lua [hunk 6]
```diff
@@ -4494,8 +4896,9 @@ OP.Global.Locales.Languages["it-IT"] = {
 		opacity = "Opacita'",
 		colours = "Colori",
 		press_to_access_clotheshop = "Premi ~INPUT_CONTEXT~ per personalizzare il personaggio.",
-		clothing_store = "Negozio Vestiti",
-		finish_creation = "Premi ~g~G ~w~ per completare la personalizzazione del personaggio. Non puoi annullare questa azione.",
+		changing_only_clotheshop = "Fai /outfit per cambiare il tuo outfit.",
+		clothing_store = "Clothing Store",
```
Comment (InZidiuZ) [inline languages/it-IT.lua:4900]: English.

File: languages/it-IT.lua [hunk 7]
```diff
@@ -4236,6 +4626,11 @@ OP.Global.Locales.Languages["it-IT"] = {
 		retrieved_vehicle_logs_title = "Veicolo recuperato",
 		retrieved_vehicle_logs_details = "${consoleName} recuperato veicolo con targa `${plate}` per $250.",
 
+		vehicle_weight = "Weight: ${weight}",
+		last_garage_letter = "Last - Garage ${letter}",
+		last_garage_impound = "Last - Impound Lot",
+		no_last_garage_letter = "No Last Garage",
```
Comment (InZidiuZ) [inline languages/it-IT.lua:4632]: English.

File: languages/it-IT.lua [hunk 8]
```diff
@@ -4068,6 +4450,14 @@ OP.Global.Locales.Languages["it-IT"] = {
 		delivery_blip = "Consegna Burger Shot"
 	},
 
+	duty = {
+		toggle_duty_status_no_permissions = "Player attempted to toggle on duty status via command without proper permissions.",
+
+		duty_status_on = "Successfully went on duty.",
+		duty_status_off = "Successfully went off duty.",
+		duty_status_failed = "Failed to toggle on duty status."
```
Comment (InZidiuZ) [inline languages/it-IT.lua:4458]: English.

**Review by InZidiuZ — CHANGES_REQUESTED** [review]
There are more issues but I don't want to manually go through this as they're all the same, repeating issues. Please review your own PRs as well for simple mistakes like these.

**Comment by InZidiuZ** [comment]
I am about to close this PR because of how polluted it has gotten. PLEASE, click [this link](https://github.com/InZidiuZ/op-framework-public/pull/123/files) and go over all the changes manually. You still have countless of issues that I have commented on numerous times now.

**Comment by InZidiuZ** [comment]
Create a new, cleaned PR. This one is too polluted.

**Pull request closed without merging**